B827EBFFFE494323.json

```diff
@@ -0,0 +1,18 @@
+{ 
+	"gateway_conf": { 
+		"gateway_ID": "B827EBFFFE494323", 
+		"servers": [ 
+			{ 
+				"server_address": "ttn-router-brazil", 

```

Comment:
This is not a valid router address, check the list of valid routers: https://www.thethingsnetwork.org/docs/gateways/packet-forwarder/semtech-udp.html#router-addresses